src/pages/tutorials/introduction-to-layer-types.md

```diff
@@ -70,8 +70,10 @@ L.esri.dynamicMapLayer({
 ```
 
 ### What does the data look like?
-When you request a Dynamic Map Layer from an ArcGIS Server the response might look something like 
-![Map tile](https://services.arcgisonline.com/arcgis/rest/directories/arcgisoutput/Specialty/Soil_Survey_Map_MapServer/_ags_map7ec03f83fc184c65b3e98b240810d139.png)
+When you request a Dynamic Map Layer from an ArcGIS Server the response might look something like the following:
+
+![Map tile](https://server.arcgisonline.com/arcgis/rest/services/Specialty/Soil_Survey_Map/MapServer/tile/8/95/65)
```

Comment:
this borders on the pedantic, but i think it'd be ideal to use an actual dynamic map service export request in the example.

https://services.arcgisonline.com/arcgis/rest/services/Specialty/Soil_Survey_Map/MapServer/export?bbox=-15214026.109881485,1868732.467515991,-7337954.715376922,8365268.375529694&size=600,200&dpi=96&format=png24&transparent=true&bboxSR=3857&imageSR=3857&f=image